apps/src/applab/DesignToolbox.jsx

```diff
@@ -100,6 +100,12 @@ export default class DesignToolbox extends React.Component {
           elementType={'SLIDER'}
           handleDragStart={this.props.handleDragStart}
         />
+        <DesignToolboxElement
+          imageUrl={IMAGE_BASE_URL + 'camera.png'}
+          desc={'Photo Chooser'}
+          elementType={'PHOTO_CHOOSER'}
+          handleDragStart={this.props.handleDragStart}
+        />
```

Comment:
2. This one's more of a NIT. It looks like the camera icon we use in the toolbox is different from the one we use on the app screen. But other examples (i.e. image, canvas, and graph) use the same or very similar icons. Would it be possible to use the same icon in both places?
<img src="https://user-images.githubusercontent.com/8324574/81615786-061b0400-9397-11ea-9306-e9797fd90f21.png" width="250"/>